tools/fp20compiler/ts1.0_inst_list.cpp

```diff
@@ -38,6 +38,14 @@ InstList& InstList::operator+=(InstPtr t)
     return *this;
 }
 
+static unsigned int float_to_uint(float value)

```

Comment:
nit: to me, it appears that the naming scheme for functions is CamelCase in this project. So it should be `FloatToUint` (or similar).

Reply:
Are the new changes acceptable? (Same for below review comment)